docs/get-started/going-live.md

```diff
@@ -67,6 +67,27 @@ yarn webiny deploy api --env=dev
 
 Since this is a new instance of the `api` stack, it will take some time to deploy. Wait for the stack deployment to finish, before moving to the next step.
 
+### 2.1 Deploy specific `api` resource
+
+From the root of your project, run the following command:
+
+```
+yarn webiny deploy api <resource-name> --env=dev
+```
+
+for example, running this command will only deploy `securityGraphQL` resource
+
+```
+yarn webiny deploy api securityGraphQL --env=dev
+```
+
+If you want to deploy more resources just add them to the list, for example
+
+```
+yarn webiny deploy api securityGraphQL filesManage databaseProxy --env=dev
+```
+> To see the list of all available resources, checkout `resources.js` file inside api folder
```

Comment:
To see the list of all available resources, checkout `resources.js` file inside api folder.